tools/depth-quality/depth-quality-model.cpp

```diff
@@ -890,6 +904,47 @@ namespace rs2
             if (save)
             {
                 _depth_sensor_model->ui = _depth_sensor_model->last_valid_ui = prev_ui;
+
```

Comment:
This seem unnatural - can it be refactored into common code?

Reply:
Yes , I refactored it and pushed the change